domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -2447,6 +2453,11 @@ public ResultsHelper(SQLRowStream sqlRowStream, Class<T> clazz) {
     return results;
   }
 
+  private <T> int getResultListRowCounts(List<T> list) {
+    return (int) list.stream().filter(e -> !(e instanceof Facet)
+      && !((e instanceof Map) && ((Map) e).containsKey("count"))).count();
+  }
+
```

Comment:
```suggestion
  /**
   * @return number of list entries excluding the Facet count and total count entries
   */
  @SuppressWarnings("rawtypes")
  private <T> int getResultListRowCounts(List<T> list) {
    return (int) list.stream()
        .filter(e -> !(e instanceof Facet) &&
                     !((e instanceof Map) &&
                         ((Map) e).size() == 1) && ((Map) e).containsKey(COUNT_FIELD))
        .count();
  }

```
* Add javadoc
* Suppress rawtypes warning
* Replace String constant
* Line wrapping that supports understanding
* Ensure map.size() == 1 because there may be data records that contain a count element.
* Can you add `"count":9` to each mockdata/poLines.json line to catch this?